Let and Const

```diff
@@ -0,0 +1,9 @@
+let foodItems = ["Pizza, Burgers, Tenders"];
+ const MENU = "On our meny we have ${foodItems)!";
```

Comment:
This will not work because you're not using backticks. This should be:

```
const MENU = `On our menu we have ${foodItems}!`;
```

Even with this, we are trying to interpolate an Array. String interpolation accepts strings or values that can be forced to a string.